apps/desktop-wallet/src/App.tsx

```diff
@@ -114,17 +117,16 @@ const App = () => {
 
       dispatch(localStorageGeneralSettingsMigrated(generalSettings))
       dispatch(localStorageNetworkSettingsMigrated(networkSettings))
-    } catch (e) {
-      console.error(e)
-      posthog.capture('Error', { message: 'Local storage data migration failed' })
+    } catch (error) {
+      sendAnalytics({ type: 'error', error, message: 'Local storage data migration failed' })
```

Comment:
We should probably remove the exception error from here.

Reply:
Yes, agreed.